### PR TITLE
Remove base-builder hardcode from benchmarks.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes https://github.com/google/fuzzbench/issues/1085

### CODE REVIEW

File: fuzzers/klee/builder.Dockerfile [hunk 1]
```diff
@@ -67,9 +65,7 @@ RUN git clone https://github.com/lmrs2/klee.git /klee && \
     wget -O tools/ktest-tool/ktest-tool https://raw.githubusercontent.com/lmrs2/klee/debug/tools/ktest-tool/ktest-tool
 
 # The libcxx build script in the KLEE repo depends on wllvm:
-# We'll upgrade pip first to avoid a build error with the old version of pip.
-RUN pip install --upgrade pip
```
Comment (jonathanmetzman) [inline fuzzers/klee/builder.Dockerfile:71]: Why are we making this change?

File: fuzzers/klee/builder.Dockerfile [hunk 2]
```diff
@@ -24,8 +24,7 @@ RUN apt-get update -y && \
 # Install KLEE dependencies.
 RUN apt-get install -y \
     cmake-data build-essential curl libcap-dev \
-    git cmake libncurses5-dev python-minimal \
```
Comment (jonathanmetzman) [inline fuzzers/klee/builder.Dockerfile:27]: Why are we making this change?

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: fuzzers/klee/builder.Dockerfile (hunk 2 above)
Comment (inferno-chromium) [reply fuzzers/klee/builder.Dockerfile:27]: So, this installs python2 on top of already existing python3. Now pip install does not support python2 anymore, so pip upgrade in command above upgrades to pip 21.3.4, which then fails with

```
#13 [ 9/14] RUN pip install wllvm
#13 sha256:bc65c9f0280549ac4d0c50639cdb9234c10d5b9f7b1a7fd509a6310a4c96881b
#13 0.290 Traceback (most recent call last):
#13 0.290   File "/usr/local/bin/pip", line 7, in <module>
#13 0.290     from pip._internal.cli.main import main
#13 0.290   File "/usr/local/lib/python2.7/dist-packages/pip/_internal/cli/main.py", line 60
#13 0.291     sys.stderr.write(f"ERROR: {exc}")
#13 0.291                                    ^
#13 0.292 SyntaxError: invalid syntax
#13 ERROR: executor failed running [/bin/sh -c pip install wllvm]: exit code: 1
```

There is no point of leaving python2 there, and to fix exception, just install wllvm using pip3.

File: fuzzers/klee/builder.Dockerfile (hunk 1 above)
Comment (inferno-chromium) [reply fuzzers/klee/builder.Dockerfile:71]: See comment below.

**Pull request merged**